### DISCUSSION TITLE
Dashboard example

### DISCUSSION THREAD

**Discussion by ericoporto** [community/community#43342] [Repositories]
### Select Topic Area

Show & Tell

### Body

I found this interface for GitHub PRs, this kind of interface where you can see all things that are happening in your repositories at the same time is the kind of thing GitHub could do better. It would be useful to include issues too and possibly discussions.

https://www.backprop.io/

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ericoporto ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
